### PR TITLE
Add thread pool metrics uts

### PR DESCRIPTION
**Pull request by namelessssssssssss**
## What is the purpose of the change
Provide some unit test in dubbo-metrics. Including thread pool metrics , etc.

## Brief changelog
* Add DefaultDubboClientObservationConventionTest in dubbo-metrics-api 
* Add DefaultDubboServerObservationConventionTest in dubbo-metrics-api
* Add DefaultMetricsServiceTest in dubbo-metrics-default
* Add ThreadPoolMetricsSamplerTest in dubbo-metrics-default 
* Add RegistryStatCompositeTest#testCalcServiceKeyRt() in dubbo-metrics-registry

### CODE REVIEW

**Comment by namelessssssssssss** [comment]
@AlbumenJ @songxiaosheng  PTAL :)

File: dubbo-metrics/dubbo-metrics-api/src/test/java/org/apache/dubbo/metrics/observation/DefaultDubboClientObservationConventionTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import io.micrometer.common.KeyValues;
+import org.apache.dubbo.metrics.observation.utils.ObservationConventionUtils;
+import org.apache.dubbo.rpc.Invoker;
+import org.apache.dubbo.rpc.RpcInvocation;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+
+@SuppressWarnings("deprecation")
```
Comment (songxiaosheng) [inline dubbo-metrics/dubbo-metrics-api/src/test/java/org/apache/dubbo/metrics/observation/DefaultDubboClientObservationConventionTest.java:27]: Why add SuppressWarnings

**Review by songxiaosheng — COMMENTED** [review]
@namelessssssssssss

**Comment by songxiaosheng** [comment]
wait  me  look  obser starter

**Comment by songxiaosheng** [comment]
@namelessssssssssss  please resolve conflicts

**Review by AlbumenJ — CHANGES_REQUESTED** [review]
Looks this PR is empty

**Pull request closed without merging**